MovieApp/PresentationLayer/Components/MoviePoster/MoviePoster+Design.swift

```diff
@@ -0,0 +1,50 @@
+import UIKit
+
+extension MoviePoster: ConstructViewsProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+    }
+
+    func createViews() {
+        posterImage = UIImageView(image: UIImage(named: "IronMan1"))
+        addSubview(posterImage)
+
+        favoriteButton = UIButton()
+        favoriteButton.addTarget(self, action: #selector(toggleFavorite), for: .touchUpInside)
+        addSubview(favoriteButton)
+
+        bringSubviewToFront(favoriteButton)
+    }
+
+    func styleViews() {
+        layer.cornerRadius = cornerRadius
+        clipsToBounds = true
+
+        favoriteButton.backgroundColor = UIColor(named: "DarkBlue")?.withAlphaComponent(0.6)
+        favoriteButton.tintColor = .white
+        favoriteButton.setImage(UIImage(named: "Favorites-outline"), for: .normal)
+    }
+
+    func roundButton() {
+        favoriteButton.layer.cornerRadius = favoriteButton.frame.height * 0.5
+    }
+
+    func defineLayoutForViews() {
+        posterImage.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+
+        favoriteButton.snp.makeConstraints {
+            $0.top.leading.equalToSuperview().inset(10)
+            $0.width.height.equalTo(buttonSize)
```

Comment:
Define `let buttonSize = CGSize(width: 32, height: 32)` and then write `$0.size.equalTo(buttonSize)`